feat(changelog): transactional-email-added-automatic-blocklist-for-bette 2025-01-29 (#4302)
scaleway/docs-content: changelog/january2025/2025-01-29-transactional-email-added-automatic-blocklist-for-bette.mdx

---

> title:  Automatic Blocklist for better deliverability

Reviewer: ```suggestion
title:  Automatic blocklist for better deliverability
```

status: added
author:

>     fullname: 'Join the #transactional_email channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #transactional-email channel on Slack.'
```

>     fullname: 'Join the #transactional_email channel on Slack.'

Reviewer 2: ```suggestion
    fullname: 'Join the #transactional-email channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2025-01-29
category: managed-services
product: transactional-email
---

Optimize your deliverability and improve your reputation score with the automatic blocklist! Addresses that consistently generate hard bounces (mailbox not found, mailbox full) are automatically blocked to prevent them from affecting your email performance.

> 📖 Learn more 👉 (link doc)

Reviewer 2: Link missing here cc @nerda-codes

> 
> Optimize your deliverability and improve your reputation score with the automatic blocklist! Addresses that consistently generate hard bounces (mailbox not found, mailbox full) are automatically blocked to prevent them from affecting your email performance.
> 
> 📖 Learn more 👉 (link doc)
> 

Reviewer: ```suggestion

Optimize your deliverability and improve your reputation score with our new automatic blocklist feature. Addresses that consistently generate hard bounces (mailbox not found, mailbox full) are automatically blocked to prevent them from affecting your email performance.

Find out about Transactional Email's blocklist feature in the [dedicated documentation](LINK).
```
